src/drivers/ins/ilabs/module.yaml

```
@@ -0,0 +1,22 @@
+module_name: InertialLabs
+
+serial_config:
+    - command: ilabs start -d ${SERIAL_DEV}
+      port_config_param:
+        name: SENS_ILABS_CFG
+        group: Sensors
+
+parameters:
+    - group: Sensors
+      definitions:
+
+        ILABS_MODE:
+            description:
+                short: InertialLabs driver mode
+                long: INS or sensors
```

The short is longer than the long. That wouldn't be a problem, but I don't know what this is for.

Here's a guess

```suggestion
                short: InertialLabs INS sensor mode configuration
                long: Configures whether the driver outputs only raw sensor output (the default), or additionally supplies INS data such as position and velocity estimates.
```

---

Thanks for the text example! Fixed